applications/serial_lte_modem/doc/DFU_AT_commands.rst

```diff
@@ -159,12 +159,14 @@ Response syntax
 
 ::
 
-  XDFUSIZE: <file_size>,<download_size>
+  XDFUSIZE: <file_size>,<download_size>,<crc32_checksum>
 
 * The ``<file_size>`` is an integer.
   It indicates the size of the DFU image file.
 * The ``<download_size>`` is an integer.
   It indicates the size that has been downloaded so far.
+* The ``<crc32_checksum>`` is an 32bit unsigned integer.
+  It indicates the IEEE CRC32 of image content that has been downloaded so far.
```

Comment:
Maybe not "so far" but the final CRC as this AT command is expected to be issued after 100% downloaded. Also this command is not required for legacy DFU. Recommend to describe the usage of this command in further details